uavcan/internet/README.md

```diff
@@ -0,0 +1,16 @@
+Internet/LAN forwarding protocol
+================================
+
+Data types defined in this namespace can be used to establish direct connectivity between arbitrary
+local UAVCAN nodes and hosts on the Internet or a local area network (LAN) by means of so called
+*modem nodes*.
+Normally, a modem node would be implemented on an on-board cellular, RF, or satellite communication
+hardware, such as an on-board telemetry modem.
+
+## Application examples
+
+* Direct telemetry transmission from UAVCAN nodes with no dependency on external means of broadcasting.
```

Comment:
Bad wording. What I meant is that one can send data out using a well-defined UAVCAN interface rather than relying on the autopilot or some other highly-capable node to tunnel stuff outside for them in an application-specific non-cross-vendor-compatible way. I have fixed this.